runtime/src/main/scala/akka/grpc/scaladsl/ServiceHandler.scala

```diff
@@ -29,31 +27,13 @@ object ServiceHandler {
   private[grpc] val isGrpcWebRequest: jmodel.HttpRequest => Boolean = matchesVariant(
     Set(GrpcProtocolWeb, GrpcProtocolWebText))
 
-  /**
-   * This is an alias for handler
-   */
   def concatOrNotFound(
       handlers: PartialFunction[HttpRequest, Future[HttpResponse]]*): HttpRequest => Future[HttpResponse] =
-    handler(handlers: _*)
-
-  /**
-   * Creates a `HttpRequest` to `HttpResponse` handler for gRPC services that can be used in
-   * for example `Http().bindAndHandleAsync` for the generated partial function handlers:
-   *  - The generated handler supports the `application/grpc` media type.
-   *  - If the request is for a handled service, but with an invalid media type, then a _415: Unsupported Media Type_ response is produced.
-   *  - Otherise if the request is not handled by one of the provided handlers, a _404: Not Found_ response is produced.
-   */
-  def handler(handlers: PartialFunction[HttpRequest, Future[HttpResponse]]*): HttpRequest => Future[HttpResponse] = {
```

Comment:
Is that one only internal so that it can just be removed?

Reply:
No, this it public API in 0.8